app/models/solidus_stripe/gateway.rb

```diff
@@ -182,5 +176,26 @@ def create_stripe_payment_intent(source, order, stripe_payment_intent_options:)
     def capture_stripe_payment_intent(stripe_payment_intent_id)
       request { Stripe::PaymentIntent.capture(stripe_payment_intent_id) }
     end
+
+    def check_given_amount_matches_order_total(amount_in_cents, options)
```

Comment:
There's no mention of the order here, thoughts on just stating that we _check given amount against the payment intent amount_?

Reply:
Makes sense. Changed!